website/server/middlewares/blocker.js

```diff
@@ -0,0 +1,69 @@
+import nconf from 'nconf';
+import {
+  Forbidden,
+} from '../libs/errors';
+import { apiError } from '../libs/apiError';
+import { model as Blocker } from '../models/blocker';
+
+// Middleware to block unwanted IP addresses
```

Comment:
This comment is out of date, yes? Since now this middleware handles more than just IP addresses.